build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
I like the idea, although I'm hesitant to add to the usage if I can help it. Would it be acceptable to check for the currently-restored-package and only call restore on it if the version has changed? That doesn't fully address the offline scenario (e.g. when doing `git checkout` to an unbuilt but already-fetched branch), but if the right optdata is missing then your release builds won't be optimized the way they are supposed to be anyway.

I can also make it restore conditionally, only when doing a release build, since we don't consume optdata during debug builds. Would this be useful?